rtl/eclrtl/eclinclude4.hpp

```diff
@@ -46,7 +46,7 @@
 #include <math.h>
 
 #define CHEAP_UCHAR_DEF
-#if __WIN32
+#if __WIN32 || _WIN64
```

Comment:
This shouldn't be needed - _WIN32 should be defined in windows 32 and 64 bit.

However this test is wrong.  It should really be 
#ifdef _WIN32